server/gokbg3/grails-app/migrations/changelog-2021-03-31.groovy

```diff
@@ -0,0 +1,59 @@
+
+databaseChangeLog = {
+
+    changeSet(author: "djebeniani (generated)", id: "1617190309675-1") {
+        grailsChange {
+            change {
+
+                def tippsWithout = sql.rows("select *\n" +
+                        "from title_instance_package_platform as tipp where\n" +
+                        "not exists(select * from tippcoverage_statement where owner_id = tipp.kbc_id)\n" +
+                        "and (tipp_start_date is not null or\n" +
+                        "     tipp_start_issue is not null or\n" +
+                        "     tipp_start_volume is not null or\n" +
+                        "     tipp_end_date is not null or\n" +
+                        "     tipp_end_issue is not null or\n" +
+                        "     tipp_end_volume is not null or\n" +
+                        "     tipp_embargo is not null or\n" +
+                        "     tipp_coverage_depth is not null or\n" +
+                        "     tipp_coverage_note is not null)")
+
+                tippsWithout.each {
+
+                    sql.execute("""insert into tippcoverage_statement(id, version, tipp_start_date, tipp_start_volume, tipp_start_issue, tipp_end_date, tipp_end_volume, tipp_end_issue, tipp_embargo, tipp_coverage_note, tipp_coverage_depth, owner_id) values
```

Comment:
id is not needed (see also nextval in line 24) since it is auto-increment (or at least should be)